src/excmds.ts

```diff
@@ -928,28 +933,27 @@ export function addJump(scrollEvent: UIEvent) {
     }
     const pageX = scrollEvent.pageX
     const pageY = scrollEvent.pageY
-    // Get config for current page
-    curJumps().then(alljumps => {
+    // Prevent pending jump from being registered
+    clearTimeout(JUMP_TIMEOUTID)
+    // Schedule the registering of the current jump
+    JUMP_TIMEOUTID = setTimeout(async () => {
+        // Get config for current page
+        const alljumps = await curJumps();
         const jumps = alljumps[getJumpPageId()]
```

Comment:
Would it make sense to have something like
```ts
let local_timeoutid = setTimeout(async () => {
    const alljumps = await curJumps();
    if (local_timeoutid != JUMP_TIMEOUTID) {
        return;
    }
    ...
}, config.get("jumpdelay"));
JUMP_TIMEOUTID = local_timeoutid;
```
To protect ourselves against race conditions when `curJumps()` somehow manages to take more than `jumpdelay` milliseconds to complete?

Reply:
Possibly, but now that we're no longer hammering `curJumps()`, the call takes around 20ms for me. Down from 200-5000ms before this change.

Reply:
I don't believe it's going to be a problem, but I implemented your suggestion to be extra defensive